flocker/node/_container.py

```diff
@@ -713,5 +716,7 @@ def calculate_changes(self, desired_configuration, current_cluster_state,
         if start_restart:
             phases.append(in_parallel(changes=start_restart))
 
-        return sequentially(changes=phases,
-                            sleep_when_empty=timedelta(seconds=1))
+        return sequentially(
+            changes=phases,
+            sleep_when_empty=NOOP_SLEEP_TIME
```

Comment:
Yeah, sorry. My mistake. I'll leave it for now to avoid having to re-run the builds.